tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -652,6 +652,23 @@ EOT
 		make
 		install -o root -g root -m 0755 umoci "${ROOTFS_DIR}/usr/local/bin/"
 		popd
+
+		# Temp PoC code: Add image signature verification artifacts into rootfs
+		rootfs_quay_verification_directory="/etc/containers/quay_verification"
+		dev_verification_directory="${script_dir}/signed-container-artifacts"
+		mkdir -p "${ROOTFS_DIR}${rootfs_quay_verification_directory}/signatures"
+		cp "${dev_verification_directory}/public.gpg" "${ROOTFS_DIR}${rootfs_quay_verification_directory}/public.gpg"
+		cp "${dev_verification_directory}/quay_policy.json" "${ROOTFS_DIR}${rootfs_quay_verification_directory}/quay_policy.json"
```

Comment:
Please use `install(1)` with explicit user/group and perms options to avoid any unpleasant surprises ;)

Reply:
I've switched them to `install -o root -g root -m 0644` which I hope will be okay.